app.py

```diff
@@ -130,56 +138,68 @@ def _end_bor_chain_callback(result, start_time: float):
     return {"result": result, "execution_time_seconds": total_time}
 
 
-@app.task(serializer="json", queue="rstuf_internals")
+@app.task(serializer="json", queue="rstuf_internals", task_time_limit=TASK_TIME_LIMIT , task_soft_time_limit=TASK_SOFT_TIME_LIMIT)
```

Comment:
this task is a chain task from `bump_online_roles`.
I think we should use the same config here, see my comment on `bump_online_roles` -- I think it should part of a specific change from there.